### ISSUE TITLE
Build failure: orca-slicer-2.2.0

### ISSUE THREAD

**Issue by Kabbone** [NixOS/nixpkgs#360342]
## Steps To Reproduce

1. build *orca-slicer-2.2.0*

## Build log

<details>

<summary>Build Log</summary>

```
Running phase: unpackPhase
unpacking source archive /nix/store/18bqykcc5r7qyhq6qzd0v2n1xz4h658d-source
source root is source
Running phase: patchPhase
applying patch /nix/store/7jq52w8wjqykpiycbvhy1qkfw9rfr339-0001-not-for-upstream-CMakeLists-Link-against-webkit2gtk-.patch
patching file src/CMakeLists.txt
Hunk #1 succeeded at 188 with fuzz 1 (offset 13 lines).
applying patch /nix/store/52sjhmzwi4jmfmbwr9ckzvjwp1x6ailz-dont-link-opencv-world-orca.patch
patching file src/libslic3r/CMakeLists.txt
Hunk #1 succeeded at 577 with fuzz 1 (offset 4 lines).
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LOCALEDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/share/doc/OrcaSlicer -DCMAKE_INSTALL_INFODIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/ranlib -DCMAKE_AR=/nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/c9zdxj8ap84v6wcm51gff5wk306yklam-orca-slicer-2.2.0 -DSLIC3R_STATIC=0 -DSLIC3R_FHS=1 -DSLIC3R_GTK=3 -DFLATPAK=1 -DBBL_RELEASE_TO_PUBLIC=1 -DBBL_INTERNAL_TESTING=0 -DDEP_WX_GTK3=ON -DSLIC3R_BUILD_TESTS=0 -DCMAKE_CXX_FLAGS=-DBOOST_LOG_DYN_LINK
-- The C compiler identification is GNU 13.3.0
-- The CXX compiler identification is GNU 13.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- SLIC3R_ASAN: OFF
-- SLIC3R_BUILD_SANDBOXES: OFF
-- SLIC3R_BUILD_TESTS: 0
-- SLIC3R_FHS: 1
-- SLIC3R_GTK: 3
-- SLIC3R_GUI: ON
-- SLIC3R_MSVC_COMPILE_PARALLEL: ON
-- SLIC3R_MSVC_PDB: ON
-- SLIC3R_PCH: ON
-- SLIC3R_PERL_XS: OFF
-- SLIC3R_PROFILE: OFF
-- SLIC3R_STATIC: 0
-- SLIC3R_WX_STABLE: OFF
-- SLIC3R_STATIC: 0
-- CMAKE_PREFIX_PATH: (default)
-- PROJECT_SOURCE_DIR: /build/source
-- CMAKE_MODULE_PATH: /build/source/cmake/modules/
-- Found PkgConfig: /nix/store/f1gamyykr0f8n4afy910ass2jx2g9zkv-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found DBus: /nix/store/qn4ghwx8p9fah3xa0byql0374szks88i-dbus-1.14.10-dev/include/dbus-1.0;/nix/store/946m0cmd00hcw2s6slzp5qc01krgrzks-dbus-1.14.10-lib/lib/dbus-1.0/include
-- LIBDIR: /build/source/src
-- LIBDIR_BIN: /build/source/build/src
CMake Warning (dev) at CMakeLists.txt:415 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /nix/store/l0cga543mgm5xwhjwwl9v8dhcp7nin2h-boost-1.80.0-dev/lib/cmake/Boost-1.80.0/BoostConfig.cmake (found suitable version "1.80.0", minimum required is "1.66") found components: system filesystem thread log locale regex chrono atomic date_time iostreams program_options
-- TBB found in /nix/store/agr2fjwfr5n4w468f1ha347xbwva4md1-tbb-2021.11.0-dev/lib/cmake/TBB
-- Found OpenSSL: /nix/store/5dlhinba4wbr5ijy0vcmyzh4i678z94w-openssl-3.3.2/lib/libcrypto.so (found version "3.3.2")
-- Found CURL: /nix/store/lz8snrzdzwkcbdjccqxd2r4f792cnwsg-curl-8.11.0/lib/libcurl.so (found version "8.11.0") 
-- Found ZLIB: /nix/store/y6bcc1vzg315zzzpir608zkhnmvp49kc-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Found Eigen3: /nix/store/7v06hdw6pg1j2g2b0nvgamlqhh80430s-eigen-3.4.0/include/eigen3 (Required is at least version "3.3")
-- Found EXPAT: /nix/store/m761lp9x14i33hb3q7a8wm52rsnk6ab1-expat-2.6.4/lib/libexpat.so (found version "2.6.4")
-- Found PNG: /nix/store/sgbfxn8byga13zwyxhpcyxds7nfm28w9-libpng-apng-1.6.43/lib/libpng.so (found version "1.6.43")
-- Found OpenGL: /nix/store/ymhcg6x5jrw3hx8ik1cji6awiybgp9f7-libglvnd-1.7.0/lib/libOpenGL.so
GLEW_ROOT: 
-- Found GLEW: /nix/store/g2jjj2jswml9244nds6zn4q8d95pdarq-glew-2.2.0-dev/include (found version "2.2.0")
-- Found NLopt in '/nix/store/9jdbkqsp7r4kf8rhr59x3wc542yrp5sy-nlopt-2.7.1'.
-- Using NLopt include directory '/nix/store/9jdbkqsp7r4kf8rhr59x3wc542yrp5sy-nlopt-2.7.1/include'.
-- Using NLopt library '/nix/store/9jdbkqsp7r4kf8rhr59x3wc542yrp5sy-nlopt-2.7.1/lib/libnlopt.so'.
CMake Deprecation Warning at cmake/modules/FindOpenVDB.cmake:126 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
Call Stack (most recent call first):
  CMakeLists.txt:602 (find_package)


-- Found OpenVDB: /nix/store/0j78d9b59z2g0srmxplsbgpvcxss43lh-openvdb-12.0.0-dev/include (found suitable version "12.0.0", minimum required is "5.0") found components: openvdb
-- OpenVDB ABI Version: 12
-- TBB found in /nix/store/agr2fjwfr5n4w468f1ha347xbwva4md1-tbb-2021.11.0-dev/lib/cmake/TBB
CMake Warning (dev) at cmake/modules/FindOpenVDB.cmake:372 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

Call Stack (most recent call first):
  CMakeLists.txt:602 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /nix/store/l0cga543mgm5xwhjwwl9v8dhcp7nin2h-boost-1.80.0-dev/lib/cmake/Boost-1.80.0/BoostConfig.cmake (found version "1.80.0") found components: iostreams system
-- find_package could not find Blosc. Using fallback blosc search...
-- Performing Test HAVE_CXX_ATOMICS64_WITHOUT_LIB
-- Performing Test HAVE_CXX_ATOMICS64_WITHOUT_LIB - Success
-- Performing Test LLVM_HAS_ATOMICS
-- Performing Test LLVM_HAS_ATOMICS - Success
-- OpenVDB libraries: /nix/store/7r6dm4xi61zrjdcwxkyx0sz6ppvs8zaq-openvdb-12.0.0/lib/libopenvdb.so
-- SPNAV library NOT found, Spacenavd not supported
CMake Deprecation Warning at src/admesh/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/boost/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/clipper/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/miniz/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/minilzo/CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Minilzo using bundled version...
CMake Deprecation Warning at src/glu-libtess/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/qhull/CMakeLists.txt:35 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- qhull Version: 7.2.0 (static linking)
CMake Deprecation Warning at src/Shiny/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/semver/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/libigl/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- IGL NOT found, using bundled version...
-- [MCUT] version: 1.2.0
-- [MCUT] MCUT_INCLUDE_DIR=/build/source/src/mcut/include
-- [MCUT] compilation_flags=-Wall;-Wextra
-- [MCUT] preprocessor_defs=-DMCUT_WITH_COMPUTE_HELPER_THREADPOOL=1
-- [MCUT] extra_libs=Threads::Threads
-- [MCUT] create target: name=mcut type=STATIC
-- [MCUT] MCUT_LIB_PATH=$<TARGET_FILE:mcut>
CMake Deprecation Warning at src/qoi/CMakeLists.txt:3 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Using header-only CGAL
-- Targeting Unix Makefiles
-- Using /nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/g++ compiler.
-- Found GMP: /nix/store/l4wqc044x1c7anj11m5zcmd6wf2mjpn3-gmp-with-cxx-6.3.0/lib/libgmp.so
-- Found MPFR: /nix/store/1iblfi4q1mbjk02z826rj3x502c6j0wk-mpfr-4.2.1/lib/libmpfr.so
CMake Warning (dev) at /nix/store/kw0wbrax22asy0vzbzbggjp199ry4v8j-cgal-5.6.2/lib/cmake/CGAL/CGAL_SetupBoost.cmake:20 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

Call Stack (most recent call first):
  /nix/store/kw0wbrax22asy0vzbzbggjp199ry4v8j-cgal-5.6.2/lib/cmake/CGAL/CGAL_SetupCGALDependencies.cmake:48 (include)
  /nix/store/kw0wbrax22asy0vzbzbggjp199ry4v8j-cgal-5.6.2/lib/cmake/CGAL/CGALConfig.cmake:168 (include)
  src/libslic3r/CMakeLists.txt:478 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /nix/store/l0cga543mgm5xwhjwwl9v8dhcp7nin2h-boost-1.80.0-dev/lib/cmake/Boost-1.80.0/BoostConfig.cmake (found suitable version "1.80.0", minimum required is "1.66")
-- Boost include dirs: /nix/store/l0cga543mgm5xwhjwwl9v8dhcp7nin2h-boost-1.80.0-dev/include
-- Boost libraries:    
-- Using gcc version 4 or later. Adding -frounding-math
-- Found OpenCV: / (found version "4.9.0") found components: core
-- Found JPEG: /nix/store/c5fqk6f08bmn8wb7xnqgciw6q9i6xvrd-libjpeg-turbo-3.0.4/lib/libjpeg.so (found version "62")
-- Adding precompiled header pchheader.hpp to target libslic3r.
CMake Deprecation Warning at src/imgui/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at src/imguizmo/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Found wxWidgets: -L/nix/store/ipsk72i76gxp952i739j2325rb6kj89j-wxwidgets-3.1.7/lib;-pthread;;;-L/nix/store/ymhcg6x5jrw3hx8ik1cji6awiybgp9f7-libglvnd-1.7.0/lib;-L/nix/store/ly5p7zz86gb19f0qk60rrylqxjs3qkc4-glu-9.0.3/lib;-lwx_baseu-3.1;-lwx_gtk3u_core-3.1;-lwx_gtk3u_html-3.1;-lwx_gtk3u_gl-3.1;-lwx_gtk3u_aui-3.1;-lwx_baseu_net-3.1;-lwx_gtk3u_media-3.1;-lwx_gtk3u_webview-3.1 (found suitable version "3.1.7", minimum required is "3.1")
-- wx-config path: /nix/store/ipsk72i76gxp952i739j2325rb6kj89j-wxwidgets-3.1.7/bin/wx-config
-- wx libs: -L/nix/store/ipsk72i76gxp952i739j2325rb6kj89j-wxwidgets-3.1.7/lib;-pthread;;;-L/nix/store/ymhcg6x5jrw3hx8ik1cji6awiybgp9f7-libglvnd-1.7.0/lib;-L/nix/store/ly5p7zz86gb19f0qk60rrylqxjs3qkc4-glu-9.0.3/lib;-lwx_baseu-3.1;-lwx_gtk3u_core-3.1;-lwx_gtk3u_html-3.1;-lwx_gtk3u_gl-3.1;-lwx_gtk3u_aui-3.1;-lwx_baseu_net-3.1;-lwx_gtk3u_media-3.1;-lwx_gtk3u_webview-3.1;/nix/store/sgbfxn8byga13zwyxhpcyxds7nfm28w9-libpng-apng-1.6.43/lib/libpng.so;/nix/store/y6bcc1vzg315zzzpir608zkhnmvp49kc-zlib-1.3.1/lib/libz.so;/nix/store/3hiw4ap7hqavshkhd4l8gzmzrdj2bg96-libtiff-4.7.0/lib/libtiff.so;/nix/store/c5fqk6f08bmn8wb7xnqgciw6q9i6xvrd-libjpeg-turbo-3.0.4/lib/libjpeg.so;/nix/store/m761lp9x14i33hb3q7a8wm52rsnk6ab1-expat-2.6.4/lib/libexpat.so;/nix/store/pacbfvpzqz2mksby36awvbcn051zcji3-glibc-2.40-36/lib/librt.so
-- Adding precompiled header pchheader.hpp to target libslic3r_gui.
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Looking for GDK_WINDOWING_WAYLAND
-- Looking for GDK_WINDOWING_WAYLAND - found
-- Looking for GDK_WINDOWING_X11
-- Looking for GDK_WINDOWING_X11 - found
-- Found GTK3: /nix/store/1yi5i1vlmqimx0qqvwdqiqhv0j65h5vb-gtk+3-3.24.43-dev/include/gtk-3.0;/nix/store/h3cmz7dg1y1rv4m059x19wvmnj626lap-at-spi2-core-2.54.0-dev/include/atk-1.0;/nix/store/pq7p4bqr2wimdp1qp01q10gbvasd55hk-glib-2.82.1-dev/include;/nix/store/pq7p4bqr2wimdp1qp01q10gbvasd55hk-glib-2.82.1-dev/include/glib-2.0;/nix/store/jq1ydifw3ip1vx94dl8w8bgvr07l6s3x-glib-2.82.1/lib/glib-2.0/include;/nix/store/x8kaq8y5mwh024vmf39n9lgbnm7hkd2c-cairo-1.18.2-dev/include/cairo;/nix/store/x0b1icbpys9gg7dhvnnjdyqp0al5r41h-freetype-2.13.3-dev/include/freetype2;/nix/store/x0b1icbpys9gg7dhvnnjdyqp0al5r41h-freetype-2.13.3-dev/include;/nix/store/kw3c67kz15lrnfnwg814jdckhh7q2iqf-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0;/nix/store/w9bhh1qd2022g14z6kkkqfh6sr2h1x5y-pango-1.54.0-dev/include/pango-1.0;/nix/store/ym8r2f74c5sq4hnwmkpv2l862fvmfjyp-harfbuzz-10.0.1-dev/include/harfbuzz
-- Found X11: /nix/store/qypxaa1d0a9vyxgcsqv2nkcdd0f8w4al-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/qrj5f1mb08k5jg7y9ikzb1njli0gvmxz-libX11-1.8.10/lib/libX11.so;/nix/store/hpd1dss7lmrywlr37vncyzbhzp6rgri5-libXext-1.3.6/lib/libXext.so
-- Looking for XOpenDisplay in /nix/store/qrj5f1mb08k5jg7y9ikzb1njli0gvmxz-libX11-1.8.10/lib/libX11.so;/nix/store/hpd1dss7lmrywlr37vncyzbhzp6rgri5-libXext-1.3.6/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Checking for module 'webkit2gtk-4.1'
--   No package 'webkit2gtk-4.1' found
CMake Error at /nix/store/4kb60dy0dg80ww8gh2av4by4vwn8l48k-cmake-3.30.5/share/cmake-3.30/Modules/FindPkgConfig.cmake:645 (message):
  The following required packages were not found:

   - webkit2gtk-4.1

Call Stack (most recent call first):
  /nix/store/4kb60dy0dg80ww8gh2av4by4vwn8l48k-cmake-3.30.5/share/cmake-3.30/Modules/FindPkgConfig.cmake:873 (_pkg_check_modules_internal)
  src/slic3r/CMakeLists.txt:666 (pkg_check_modules)


-- Configuring incomplete, errors occurred!
```

</details>

## Additional context

Seems like you forgot to add webkit2gtk-4.1 as a dependency since bambu-studio only has webkit2gtk-4.0

## Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.12-valve5, NixOS, 24.11 (Vicuna), 24.11.20241111.dc460ec`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - nixpkgs: `/nix/store/4r8s42c9mwfvgdlxv0izb1cmlzrsb5nz-source`

## Notify maintainers

@zhaofengli
@ovlach
@pinpox 
---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by azerbe** [comment]
Seems like it's already fixed: https://github.com/NixOS/nixpkgs/issues/358666

**Comment by rkliman** [comment]
Still seeing this today as well. If it's fixed by that PR, what's the way to fix this?

**Comment by Rijndael1998** [comment]
> Still seeing this today as well. @azerbe If it's fixed by that PR, what's the way to fix this on a local installation?

It's still broken for me.

**Comment by NullCub3** [comment]
@rkliman @Rijndael1998  The fix hasn't been merged into the nixos-unstable branch yet: https://nixpk.gs/pr-tracker.html?pr=360219

However it is available in nixpkgs-unstable and nixos-unstable-small, and if you use flakes you can add one of those branches as a flake input and then do something like this for a quick fix:
```nix
# Use an overlay to replace all instances of orca-slicer
nixpkgs.overlays = [
  (final: prev: { orca-slicer = inputs.nixos-unstable-small.legacyPackages.${prev.system}.orca-slicer; })
];
# Or just install the package directly:
# inputs.nixos-unstable-small.legacyPackages.${pkgs.system}.orca-slicer
```

I don't know the non-flakes method of the top of my head but the fix should be in nixos-unstable soon I would hope? No idea about stable.

**Comment by bobberb** [comment]
Now that #360219 is in unstable, who decides about backporting?

**Comment by liberodark** [comment]
That have been fixed by : 
https://github.com/NixOS/nixpkgs/pull/360219
https://github.com/NixOS/nixpkgs/pull/360738

**Comment by Scrumplex** [comment]
This issue only affected unstable and 24.11. The fix in #360219 has reached both unstable and 24.11.
